src/System.Net.Sockets/tests/FunctionalTests/AcceptAsync.cs

```diff
@@ -8,7 +11,7 @@ namespace System.Net.Sockets.Tests
 {
     public class AcceptAsync
     {
-        private const int TestPortBase = 8000;
+        private const int TestPortBase = 7000;
```

Comment:
> Some tests will always use allocated ports.

By "allocated" do you mean "chosen ahead of time"? If so, then we'll have to choose those tests very carefully or potentially disable them in CI if it's a requirement that a test must be able to run in parallel with itself (e.g. the same test may be running in multiple processes).